Onboarding/Onboarding/StepDefinitions/CertificationStepDefinitions.cs

```diff
@@ -11,105 +12,109 @@ namespace Onboarding.StepDefinitions
     [Binding]
     public class CertificationStepDefinitions : CommonDriver
     {
-        [After]
-        public void Dispose()
-        {
-            if (driver != null)
-            {
-                driver.Close();
-            }
-        }
+        //Define Pages and Objects
+        Certification CertificationObj;
+        LoginPage LoginPageObj;
 
-        Certification CertificationObj = new Certification();
         [Given(@"I signed into the portal")]
         public void GivenISignedIntoThePortal()
         {
             //Open Chrome browser
             driver = new ChromeDriver();
```

Comment:
also drive you can move to the Common class and initialise it once and use it 


namespace MarsQA.Utilities
{
    public class CommonDriver
    {
        public static IWebDriver driver;
        LoginPage loginpageObj = new LoginPage();

        [OneTimeSetUp]
        public void onTimeSetup()
        {
            // Open chrome browser
            driver = new ChromeDriver();

            // Login page object initialization and definition you dont have to do this here if you don't want too
            loginpageObj.LoginSteps(driver); // this part you can do in pages if you want or leave it here 

        }


        [OneTimeTearDown]
        public void CloseTestRun()
        {
            // Open chrome browser
            driver = new ChromeDriver();

            driver.Quit();
        }
    }
}

Reply:
this way you dont have to do new driver every where as the in the common class diver variable is static
public static IWebDriver driver;

Reply:
done as no 'static'